ts/src/main/scala/ch/epfl/ts/component/fetch/Fetch.scala

```diff
@@ -52,10 +52,16 @@ class PullFetchListComponent[T: ClassTag](f: PullFetch[T]) extends Component {
 }
 
 /* Actor implementation */
+/**
+ * To implement your own PushFetchComponent:
+ * 1. Create your class C, C extends PushFetchComponent
+ * 2. Inside your class use callback(what you want to send) 
```

Comment:
I don't think this is the best way. If you take inspiration from `PullFetchComponent`, you would see that `PullFetchComponent` is a *generic* wrapper around a `PullFetch`. It very simply makes an actor out of it, and ensures that all `PullFetch` have consistent behavior.

For example:
```scala
class TrueFxFetcher extends PullFetch[Quote] {
  // ...
}

// Usage: the Component is just a wrapper
val fetcherFx: TrueFxFetcher = new TrueFxFetcher
val fxQuoteFetcher = builder.createRef(Props(classOf[PullFetchComponent[Quote]], fetcherFx, implicitly[ClassTag[Quote]]), "trueFxFetcher")
    
```